can_tools/scrapers/official/NH/nh_demographics.py

```diff
@@ -66,7 +66,14 @@ def _get_county(variable: str, county: str):
         )
 
         engine.getWorkbook().setParameter("ShowBy", "Race/Ethnicity")
-        engine.getWorkbook().setParameter("Metric", variable)
+        # By default the parameter is set to "Total Individuals Completed Vaccine"
```

Comment:
This scraper is failing on the server but is successful locally. Locally, we were getting some warnings, which I think might be the cause of the failures on the server. 

I don't know why the behavior is different between the two--I ran updates on the Prefect VM to try to fix this, but that didn't change anything

Reply:
I'm hoping this resolves it, but I'm honestly not sure

Reply:
Do you know if prefect runs `pip install -e .` or equivalent somehow?  Maybe it is using the old `tableauscraper` dependency?

Reply:
Yeah, that's a good thought. Admittedly, I don't fully know how Prefect pulls in the changes so I'm looking for that workflow/process now. It's just kinda' always worked haha 😭

Reply:
FWIW- If we can't figure out what Prefect is (or isn't doing) we could log the tableauscraper version at runtime per https://stackoverflow.com/a/4939465/110915 to find out for sure if there's a problem.

Reply:
Looks like you might be right, if I'm understanding correctly. 

The webhook pulls in the new code and kicks off the Prefect flows here, I think

https://github.com/covid-projections/can-scrapers/blob/ba36f84d9d9806721f9b5f5b2b4ed9e086176f10/services/webhook/update_flows.sh#L3

Reply:
and it doesn't seem to update the dependencies

Reply:
ah, didn't refresh to see your comment. Logging seems like a good start

Reply:
Up to you!  It seems like you're on the right track with that script.  I'm fine with just adding a `pip install -e .` or whatever.

Reply:
I think I'll add a logger to this to check the version, and if that is the issue then continue looking at that script/the webhooks to try to figure out what's going on, if that sounds good.

Reply:
👍  Sounds great.